api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddAuthorsToInformationResourceGenerator.java

```diff
@@ -171,13 +185,51 @@ private String getN3NewPersonLastName() {
 	}
 
 	private String getN3NewPerson() {
-		return  getN3PrefixString() +
+		return  getN3PrefixString() + getDisableRealPersonPrefix() +
         "?newPerson a foaf:Person ;\n" +
         "<" + RDFS.label.getURI() + "> ?label .\n" +
         "?authorshipUri core:relates ?newPerson .\n" +
         "?newPerson core:relatedBy ?authorshipUri . ";
 	}
 
+	// Changes here for creating vcards for external authors
+	private String getN3NewPersonVCardFirstName() {
```

Comment:
The method naming here is also a bit confusing here with `getN3...`

Reply:
Yes. But it is aligned with other method names used in N3 generators.